Example/Example/Shared/Repository.swift

```diff
@@ -15,25 +15,38 @@ final class Repository {
   private let userDefaults: UserDefaults
   private let now: () -> Date
 
+  private let locale = Locale(identifier: "en_US")
+  private let environment = Environment.sandbox
+
   static let shared = Repository(apiClient: .live, userDefaults: .standard, now: Date.init)
 
-  var cachedConfiguration: Configuration {
-    let response = userDefaults.configuration.flatMap {
-      try? JSONDecoder().decode(ConfigurationResponse.self, from: $0)
-    }
+  private var fallbackConfiguration: Configuration {
+    // swiftlint:disable:next force_try
+    try! Configuration(
+      minimumAmount: nil,
+      maximumAmount: "1000.00",
+      currencyCode: "USD",
+      locale: locale,
```

Comment:
This should be the only spot that this `locale` member variable is being used so maybe we can inline it